pywinauto/unittests/test_Calendar.py

```diff
@@ -92,7 +92,28 @@ def assert_system_time_is_equal_to_current_date_time(self,systemTime, now):
         self.assertEqual(systemTime.wDay, now.day)
 
     def set_calendar_state_into_multiselect(self):
-        self.app['Common Controls Sample']['MCS_MULTISELECT'].WrapperObject().Click()         
+        self.app['Common Controls Sample']['MCS_MULTISELECT'].WrapperObject().Click()
+
+    def test_can_get_today(self):
+        """Test getting the control's today field"""
+        date = self.calendar.get_today()
+        self.assert_system_time_is_equal_to_current_date_time(date, datetime.date.today())
+
+    def test_can_set_today(self):
+        """Test setting up the control's today field"""
+        self.calendar.set_today(2016, 5, 1)
+        self.assert_system_time_is_equal_to_current_date_time(self.calendar.get_today(),
+                                                              datetime.date(2016, 5, 1))
+
+    def test_is_first_day_of_week_locale(self):
+        """Test is the today field set as LOCALE_IFIRSTDAYOFWEEK"""
+        self.assertEqual(0, self.calendar.get_first_weekday())
+
+    def test_can_set_and_get_first_day_of_week(self):
+        """Test can set and get first day of the week"""
+        self.calendar.set_first_weekday(4)
+        self.assertEqual(4, self.calendar.get_first_weekday()-65536)
```

Comment:
@vasily-v-ryabov Can you give me advice ? 
MCM_GETFIRSTDAYOFWEEK Returns a DWORD value that contains two values. The high word is a BOOL value that is nonzero if the first day of the week is set to something other than LOCALE_IFIRSTDAYOFWEEK, or zero otherwise. The low word is an INT value that represents the first day of the week, where 0 is Monday, 1 is Tuesday, and so on.

What should I return? Just the low word from 0 to 6 ?